packages/atlas/src/components/_overlays/SendTransferDialogs/SendFundsDialog.tsx

```diff
@@ -0,0 +1,195 @@
+
+type SendFundsDialogProps = {
+  onExitClick: () => void
+  accountBalance?: number
+  show: boolean
+}
+
+export const SendFundsDialog: FC<SendFundsDialogProps> = ({ onExitClick, accountBalance = 0, show }) => {
+  const [destinationAccount, setDestinationAccount] = useState<BasicMembershipFieldsFragment>()
+  const { convertToUSD } = useTokenPrice()
+  const client = useApolloClient()
+  const {
+    register,
+    reset,
+    handleSubmit,
+    watch,
+    setValue,
+    formState: { errors },
+  } = useForm<{ amount: number | null; account: string | null }>({ mode: 'onSubmit' })
+  const convertedAmount = convertToUSD(watch('amount') || 0)
+
+  useEffect(() => {
+    if (!show) {
+      reset({ amount: null, account: null })
+    }
+  }, [reset, show])
+
+  const debounceFetchMembers = useRef(
+    debouncePromise(async (val?: string) => {
+      if (!val) {
+        return
+      }
+      try {
+        const {
+          data: { memberships },
+        } = await client.query<GetMembershipsQuery, GetMembershipsQueryVariables>({
+          query: GetMembershipsDocument,
+          variables: { where: { controllerAccount_eq: val } },
+        })
+        setDestinationAccount(memberships.length ? memberships[0] : undefined)
+      } catch (error) {
+        SentryLogger.error('Failed to fetch memberships', 'WhiteListTextField', error)
+      }
+    }, 500)
+  )
+
+  const handleSendFounds = async () => {
+    const handler = await handleSubmit(() => null)
+    return handler()
+  }
+
+  const handleMaxClick = () => {
+    setValue('amount', accountBalance, { shouldTouch: true, shouldDirty: true, shouldValidate: true })
+  }
+
+  return (
+    <DialogModal
+      show={show}
+      title="Send"
+      onExitClick={onExitClick}
+      primaryButton={{ text: 'Send', onClick: handleSendFounds }}
+      secondaryButton={{ text: 'Cancel', onClick: onExitClick }}
+      additionalActionsNode={<Fee />}
+    >
+      <Text as="h4" variant="h300" margin={{ bottom: 4 }}>
+        Your channel balance
+      </Text>
+      <BalanceWrapper>
+        <JoyTokenIcon variant="gray" />
+        <Text as="p" variant="h400" margin={{ left: 1 }}>
+          {accountBalance}
+        </Text>
+      </BalanceWrapper>
+      <NumberFormat
+        as="p"
+        color="colorText"
+        format="dollar"
+        variant="t100"
+        value={convertToUSD(accountBalance) || 0}
+        margin={{ top: 1, bottom: 6 }}
+      />
+      <FormFieldsWrapper>
+        <FormField
+          label={
+            <LabelFlexWrapper>
+              <Text as="span" variant="h300" margin={{ left: 2 }}>
+                Amount to withdraw
+              </Text>
+              <Button onClick={handleMaxClick} size="medium" variant="tertiary" _textOnly>
+                Max
+              </Button>
+            </LabelFlexWrapper>
+          }
+          error={errors.amount?.message}
+        >
+          <Input
```

Comment:
I'm wondering - could we use the FormField prop `disableErrorAnimation` to not shake the input when typing(when revalidating onChange) after an unsuccessful submission? I believe that there is something like `submissionCount` or `submitted` in react hook form.  

I mean, I'm fine with leaving this as it is. Just a suggestion